src/app/user/user.service.ts

```diff
@@ -19,16 +19,44 @@ export class UserService {
   }
 
   // add User
-  async adduser(req: Request, res: Response) {
+  async addUser(req: Request, res: Response) {
     const newUser = new this.model({
       firstName: req.body.firstName,
       lastName: req.body.lastName,
       email: req.body.email,
       password: req.body.password,
-    })
+    });
     try {
       await this.model.create(newUser);
-      res.status(201).json({ message: 'User create Successfully' })
+      res.status(201).json({ message: 'User create Successfully' });
+    } catch (error) {
+      throw error;
+    }
+  }
+
+  // Update User
+  async updateUser(req: Request, res: Response) {
+    const id: any = req.params.id;
+    const updUser = {
+      firstName: req.body.firstName,
+      lastName: req.body.lastName,
+      email: req.body.email,
+      password: req.body.password
+    };
+    try {
+      await this.model.findByIdAndUpdate(updUser, id);
+      res.status(200).json({ message: 'Update User Successfully' });
+    } catch (error) {
+      throw error;
+    }
+  }
+
+  // Delete User
+  async deleteUser(req: Request, res: Response) {
+    const id: any = req.params.id;
```

Comment:
@hsedaqat1378 please don't use `any` as much as you can. use `string` in this case. using `any` is like telling typescript to go fuck yourself I'm good with js 😄